docs/reset-psw-user.rst

```diff
@@ -0,0 +1,22 @@
+=====================================
+Reset specific user password by shell
+=====================================
+
+In case you don't remember the user password it is easy to do change by CLI. For example:
+
+- Run the shell mode in Django:
+
+   ``docker-compose run --rm web ./manage.py shell``
+
+- After loading you will be in the shell mode where you can run python code:
+
+.. code-block::
+
+    from django.contrib.auth.models import User #import the class
+    users = User.objects.filter()
```

Comment:
Probably is better to replace this with a `get`.
`user = User.objects.get(email='user_email')`

Reply:
Even so, here you just change the password and not the email or username. This means that you need to remember eg the username in order to login. You can replace the email with username if you prefer. Finally, if you want to get all the users in Django, use `all()` instead of an empty filter.